### PR TITLE
Clear the Empty spaces in /chapter route

### PR DESCRIPTION
**Pull request by sirasitxp**
- [X] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [X] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [X] My pull request targets the `main` branch of Chapter.

Target for issue #1244

### CODE REVIEW

**Comment by Sboonny** [comment]
Hey, @hisnameispum because the current pull request won't support the current development for mvp, I am wondering if you can turn it into grid with 2 or more columns like mentioned in [this comment](https://github.com/freeCodeCamp/chapter/issues/1244#issuecomment-1210280089)

**Comment by hisnameispum** [comment]
> this comment

Got it!

**Comment by hisnameispum** [comment]
@Sboonny I have redesigned the layout to be more like table list. Please let me know if it works for you. 

Screen Shot 2022-08-16 at 1 29 42 PM

**Comment by Sboonny** [comment]
Sorry, looking back I was vague in my description. 

Here is a quickly made layout in the figma of how the grid can look like
https://www.figma.com/file/o3BhXYjbJHb90Rbr4hlABP/Untitled?node-id=0%3A1

you can use `Grid, GridItem` in chakra to scale it easier in the future

https://chakra-ui.com/docs/components/grid/usage

**Comment by hisnameispum** [comment]
> 

It have changed it from Flex to Grid, let me know if it works for you.

File: client/src/modules/chapters/pages/chaptersPage.tsx [hunk 1]
```diff
@@ -20,16 +30,61 @@ export const ChaptersPage: NextPage = () => {
...
+                        <Td borderWidth={2}>{chapter.name}</Td>
+                        <Td borderWidth={2}>{chapter.description}</Td>
+                      </Tr>
+                    ))}
+                  </Tbody>
+                </Table>
+              </TableContainer>
+            </GridItem>
+          </Grid>
+        </Flex>
```
Comment (Sboonny) [inline client/src/modules/chapters/pages/chaptersPage.tsx:87]:
```suggestion
      <Stack w={['90%', '90%', '60%']} maxW="600px" spacing={3} mt={10} mb={5}>
          <Heading>Chapters: </Heading>
          <Grid mt="5%" templateColumns="repeat(2, 1fr)" columnGap="5%">
          {data.chapters.map((chapter) => (
            <GridItem>
            <Heading size="md" key={chapter.id}>
              <ChapterCard key={chapter.id} chapter={chapter} />
            </Heading>
            </GridItem>
          ))}   
          </Grid>
      </Stack>
```

File: client/src/modules/chapters/pages/chaptersPage.tsx [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  TableContainer,
+  Grid,
+  GridItem,
+} from '@chakra-ui/react';
```
Comment (Sboonny) [inline client/src/modules/chapters/pages/chaptersPage.tsx:14]:
```suggestion
import { Grid, GridItem } from '@chakra-ui/react';
```

File: client/src/modules/chapters/pages/chaptersPage.tsx [hunk 3]
```diff
@@ -1,8 +1,18 @@
-import { Heading, VStack, Stack } from '@chakra-ui/layout';
+import { Heading, VStack, Stack, Flex, Text } from '@chakra-ui/layout';
```
Comment (Sboonny) [inline client/src/modules/chapters/pages/chaptersPage.tsx:1]:
```suggestion
import { Heading, VStack, Stack } from '@chakra-ui/layout';
```

**Review by Sboonny — CHANGES_REQUESTED** [review]
Thank you for the hard work, and your patience

 can you delete the two extra `.env` files, and revert the extra changes? I need something small

**Comment by hisnameispum** [comment]
> Thank you for the hard work, and your patience
> 
> can you delete the two extra `.env` files, and revert the extra changes? I need something small

Done! Let me know if you need me to do anything more.

File: client/src/modules/chapters/pages/chaptersPage.tsx [hunk 4]
```diff
@@ -1,8 +1,8 @@
 import { Heading, VStack, Stack } from '@chakra-ui/layout';
 import { NextPage } from 'next';
 import React from 'react';
+import { Grid, GridItem } from '@chakra-ui/react';
 
-import { ChapterCard } from 'components/ChapterCard';
```
Comment (Sboonny) [inline client/src/modules/chapters/pages/chaptersPage.tsx:5]:
```
import { ChapterCard } from 'components/ChapterCard';
```

**Review by Sboonny — COMMENTED** [review]
The component is used, but not imported. This is leading to error

File: client/src/modules/chapters/pages/chaptersPage.tsx (hunk 4 above)
Comment (hisnameispum) [reply client/src/modules/chapters/pages/chaptersPage.tsx:5]: I kept getting eslint error after adding the import statement back. I'll get back to work on this on Monday.

Comment (hisnameispum) [reply client/src/modules/chapters/pages/chaptersPage.tsx:5]: Screen Shot 2022-08-22 at 12 26 51 PM
Screen Shot 2022-08-22 at 12 27 39 PM

@Sboonny 
I kept getting the same issue even though i added the key prop to the outer most element, is there anything wrong with eslint version? I did some research and found them here. https://github.com/jsx-eslint/eslint-plugin-react/issues/3215

Comment (Sboonny) [reply client/src/modules/chapters/pages/chaptersPage.tsx:5]: Map is a higher function that creates an array, that react requires key for it.

Placing key in header should please eslint, if it still isn't accepting the changes, I will open the pull request in the morning, if you don't mind This.

Comment (hisnameispum) [reply client/src/modules/chapters/pages/chaptersPage.tsx:5]: @Sboonny I think I got it to work now with the new commit. Please have a look and let me know if you need anything to be done from my end.

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Pull request merged**